MEvoLibGUI/MEvoLibGUI/tasks.py

```diff
@@ -0,0 +1,37 @@
+from celery import shared_task
+from pathlib import Path
+import nextflow, os, re
+
+from MEvoLibGUI.settings import (
+    WORKFLOW_OUTPUT as wf_output,
+)
+    
+@shared_task()
+def run_workflow(workflow_path, parameters):
+    """Orders a Nextflow workflow to start, so that it can be executed asynchronously while the user keeps on
+    doing other tasks."""
+
+    workflow_error = re.compile(r"^.+ ERROR .+$")
+    task_error = re.compile(r"^.+ terminated with an error .+$")
```

Comment:
I would suggest to make these two constants.

Reply:
Gotcha